modules/shortcodes/polldaddy.php

```diff
@@ -573,7 +573,7 @@ function polldaddy_link( $content ) {
 	add_filter( 'the_content_rss', 'polldaddy_link', 1 );
 
 	/** This filter is documented in modules/shortcodes/youtube.php */
-	if ( apply_filters( 'jetpack_comments_allow_oembed', get_option( 'embed_autourls' ) ) ) {
+	if ( apply_filters( 'jetpack_comments_allow_oembed', true ) ) {
```

Comment:
I'm not against sticking with the `get_option`, but it seems that WordPress is already [attaching a filter to that](https://github.com/WordPress/WordPress/blob/master/wp-includes/default-filters.php#L333). Do you think there could be filters with a higher priority attached?